pkg/nmi/server/windowsServer.go

```diff
@@ -0,0 +1,83 @@
+// +build windows
+
+package server
+
+import (
+	"fmt"
+	"sync"
+
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/klog"
+)
+
+type WindowsServer struct {
+	Server *Server
+}
+
+// Run runs the specified Server.
+func (s *WindowsServer) Run() error {
+
+	var wg sync.WaitGroup
+
+	wg.Add(1)
```

Comment:
This will fire one thread right, so do you still need thread synchronization using waitgroup?

Reply:
I think you are right. No need to have sync.WaitGroup here. Updated the code.